architecture/tube.go

```diff
@@ -213,3 +214,33 @@ func (tube *Tube) Bury(command *Command) {
 		command.Err = errors.New(NOT_FOUND)
 	}
 }
+
+func (tube *Tube) Kick(command *Command) {
+	amount := 0
+	bound, err := strconv.Atoi(command.Params["bound"])
+	if err != nil {
+		// handle non-integer number of jobs to kick
```

Comment:
@marekyggdrasil Your idea makes sense. I was additionally thinking the `Params` array can be made in to a map. But that feels like unnecessary complication, isn't it? 
I will approve and merge this pull req now. I created a separate issue #67 to discuss and work on validation for parameters.